cdap-source-control/src/main/java/io/cdap/cdap/sourcecontrol/RepositoryManager.java

```diff
@@ -174,7 +175,7 @@ public static void validateConfig(SecureStore secureStore, SourceControlConfig s
    * not OK
    */
   public String commitAndPush(CommitMeta commitMeta, Path fileChanged)
-    throws NoChangesToPushException, GitAPIException, PushFailureException {
```

Comment:
There seems to be a redundant `NoChangesToPushException` in the javadoc(my mistake in last PR...) , and also pls add the new `SourceControlException`

Reply:
Done